indexer/services/comlink/public/api-documentation.md

```diff
@@ -5,10 +5,7 @@
 
 Base URLs:
```

Comment:
did we mean to update this file?

Reply:
This gets updated when you run the indexer proto gen. Something seems to be broken with that script at the moment